app/mva/fork_test.go

```diff
@@ -192,7 +192,19 @@ func (suite *ForkTestSuite) TestFork() {
 			},
 		},
 		{
-			"manual vesting account with some delegated vesting and delegated free coins", args{
+			"manual vesting account with some delegated vesting coins (2)", args{
+				copyMVA(baseMVA2),
+				[]sdk.Int{sdk.NewInt(2000000)},
+				[]sdk.Int{sdk.NewInt(1000000)},
+			},
+			errArgs{
+				true,
+				sdk.NewCoins(sdk.NewInt64Coin(common.MicroCTKDenom, 1000000)),
+				sdk.NewCoins(),
```

Comment:
I think this should be 2000000 because delegated is sum of delegation and unbonding?